modules/store/schematics/BUILD

```diff
@@ -0,0 +1,34 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
This should be added to the `packages` section in the `modules/store/BUILD` file in the `npm_package`. That way its output gets copied into the distribution.